astropy/time/config.py

```diff
@@ -0,0 +1,19 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from astropy import config as _config
+
+
+class Conf(_config.ConfigNamespace):  # noqa
```

Comment:
This is out of scope and probably an API change. Does not look related to isort.

Reply:
Can delete.
I don't thinks it's an API change for the user since the all the contents of ``config.py`` are dumped into ``__init__``. The only thing that is removed from ``__init__`` is the private variable ``_config``.

Reply:
I put this in here because then isort can sort more happily without ``E403`` errors related to splitting imports.

Reply:
But you also didn't mention you moved it. I wouldn't have known without looking at the diff. I would prefer this be a separate PR from isort. Thanks!

Reply:
Re: E403 -- Can isort ignore that for now? Or does that block this work?

Reply:
Moved to a subsequent PR.

Reply:
It doesn't block. I have to turn off `isort` for those imports anyway because they are circularly dependent.